src/download/linux.md

```diff
@@ -40,6 +40,15 @@ yay -S polymc-bin
 # latest git package:
 yay -S polymc-git
 ```
+
+If you want to use Qt 5 to build the packages instead:
+```
+# stable Qt 5 source package:
+yay -S polymc-qt5
+# latest Qt 5 git package:
+yay -S polymc-qt5-git
```

Comment:
If https://github.com/PolyMC/PolyMC/pull/922 gets merged you'll have to add polymc-qt5-bin too btw